### ISSUE TITLE
Customer error report: Linking with SAFIR_SDK_CORE_LIBRARIES (from cmake) also requires libodbc

### ISSUE THREAD

**Issue by DonOregano** [SafirSDK/safir-sdk-core#307]
Email from customer (non-relevant details redacted):

```
xxxx m.fl. kräver att vi länkar med libodbc.
Detta har vi inte behövt tidigare och tänkte kolla med er om ni vet orsaken till detta.  
xxxx har inga DB-kopplingar så jag tycker det verkar konstigt.
...
Vi har ändrat våra cmake filer till:
find_package(SafirSDKCore REQUIRED)
target_link_libraries(${PROJECT_NAME} ${SAFIR_SDK_CORE_LIBRARIES})

Är det där felet ligger?
Dvs att vi länkar med alla Core’s lib och någon av dem har ett beroende som vi måste ha med?
```

We may need to add libodbc to SAFIR_SDK_CORE_LIBRARIES, but this is a bit strange.

**Comment by DonOregano** [comment]
One solution would be to remove olib from SAFIR_SDK_CORE_LIBRARIES and add a separate variable for Olib.

But maybe we can remove Olib altogether, or at least do #219.

**Comment by DonOregano** [comment]
Erroneously closed the issue.

**Comment by DonOregano** [comment]
Still have not been able to reproduce. Waiting for input from reporter.

**Comment by DonOregano** [comment]
Fixed by #325.
